templates/layouts/Default:-petition-and-scroll-to-share-greenpeace.liquid

```diff
@@ -20,7 +20,7 @@
   </div>
 
```

Comment:
Is there a difference between the `Default Petition And Scroll To Share Greenpeace` and `Petition And Scroll To Share Greenpeace` templates? I think we should make the changes to both templates since I see a bunch of pages using the latter on production.

Reply:
Hey I checked on production to be sure, there are no pages that use this template! They all use `Petition And Scroll To Share Greenpeace` so do make sure to modify that one, as well!

Reply:
There's only one greenpeace template Tuuli

Reply:
I think then we'll need to rename this one back to the old name... I'm trying to find out why this one was renamed to begin with before doing that and potentially messing something up...